remove platform configuration settings and re-phrase (#1271)
SoftwareAG/c8y-docs: content/users-guide/enterprise-tenant-bundle/ms-data-broker.md

<a name="enabling-ms-data-broker"></a>
###  To enable the microservice-based data broker

> 1. In the {{< management-tenant >}}, navigate to **Administration** > **Tenants** > **Subtenants**, and select the tenant that will serve as the source of the data connector.

Author: @rssag Must be moved to the Ops guide

> The microservice-based data broker must be enabled from the {{< management-tenant >}}. Contact your Operations team on any further support.

Reviewer: ```suggestion
The microservice-based data broker must be enabled from the {{< management-tenant >}}. Contact your Operations team for further support.
```
